gen/suites/actor_deletion/main.go

```diff
@@ -0,0 +1,137 @@
+package main
+
+import (
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/specs-actors/actors/abi/big"
+	"github.com/filecoin-project/specs-actors/actors/builtin"
+	"github.com/filecoin-project/specs-actors/actors/runtime/exitcode"
+	"github.com/filecoin-project/test-vectors/chaos"
+	. "github.com/filecoin-project/test-vectors/gen/builders"
+	"github.com/filecoin-project/test-vectors/schema"
+)
+
+func main() {
+	g := NewGenerator()
+
+	g.Group("no_beneficiary",
+		&VectorDef{
+			Metadata: &Metadata{
+				ID:      "delete-w-zero-balance",
+				Version: "v1",
+				Desc:    "actor with zero balance is deleted, does not require beneficiary",
+			},
+			Selector:    schema.Selector{"chaos_actor": "true"},
+			MessageFunc: deleteActor,
+		},
+	)
+
+	g.Group("beneficiary",
+		&VectorDef{
+			Metadata: &Metadata{
+				ID:      "delete-w-balance-and-beneficiary",
+				Version: "v1",
+				Desc:    "actor with non-zero balance is deleted and sends funds to beneficiary",
+			},
+			Selector:    schema.Selector{"chaos_actor": "true"},
+			MessageFunc: deleteActorWithBeneficiary(big.NewInt(50), address.Undef, exitcode.Ok),
+		},
+		// TODO: uncomment when merged https://github.com/filecoin-project/lotus/pull/3479
+		// &VectorDef{
```

Comment:
```suggestion
        // It is not marked with HintInvalid because it panics entirely.
		// &VectorDef{
```

Reply:
Adding a note to our future selves because we might wonder why we didn't mark it as invalid.